react/modules/tasks/nextjs-ssr-ssg.md

```diff
@@ -0,0 +1,41 @@
+# Next.js. Server Side Rendering
+
+## Technical requirements
+
+1. Create a separate branch for this task from the previous branch task. Branch name: "nextjs-ssr-pages-api".
+2. Next.js Pages API Integration:
+   - Migrate your application from vite to next.js by using Pages folder API. You'll need to adapt existing libraries to work with next.js.
+3. Next.js App Router API Integration:
+   - Create a separate branch for this point from "nextjs-ssr-pages-api" branch. Branch name: "nextjs-ssr-app-router-api".
+   - Migrate from Pages Api to App Folder Api.
+4. Remix Integration:
+   - Create a separate branch for this point from the previous branch task. Branch name: "remix-ssr".
+   - Add Remix framework to your vite config
+   - Migrate your application to ssr with Remix
```

Comment:
```
- Update routing
```
And some more docs for routing in Remix
https://remix.run/docs/en/main/file-conventions/routes